polybar-scripts/info-timew/README.md

```diff
@@ -0,0 +1,13 @@
+# Script: timew
+
+This script displays the daily tracked time. You can click it, to start tracking, using timew.
+
+## Module
+
+```ini
+[module/timew]
+type = custom/script
+exec = ~/.config/polybar/scripts/timew.sh
+interval = 2
+click-left = "if timew;then timew stop;else timew start; fi"
```

Comment:
It would be cool if this logic would be part of the script. We could use a command line argument. If its not set print the regular output.